openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -272,10 +275,18 @@ def update_account_settings(requesting_user, update, username=None):
         if 'extended_profile' in update:
             meta = existing_user_profile.get_meta()
             new_extended_profile = update['extended_profile']
+            # We get the custom form fields by user.
+            custom_form_fields_obj, created = CustomFormFields.objects.get_or_create(user=existing_user)  # pylint: disable=W0612
```

Comment:
we could also move this to the plugin, see how @felipemontoya did it here https://github.com/eduNEXT/edx-platform/pull/138/files#diff-df4681b6dbe6bbc209ddf0f521adb0a6